.tmux.conf

```diff
@@ -90,7 +90,7 @@ wg_activated_gh_account="#(~/.config/tmux/.__activated_gh_account.sh)"
 wg_working_dir="#(~/.config/tmux/.__shorten_current_path.sh #{pane_current_path})"
 
 set -g status-left "$wg_session"
-set -g status-right "#[fg=$color_secondary]$wg_activated_gh_account #[default]| #[fg=$color_working_dir]$wg_working_dir #[default]| CPU: #{cpu_fg_color}#{cpu_percentage} #[default]| RAM: #{ram_fg_color}#{ram_percentage}  "
+set -g status-right "#{prefix_highlight} #[fg=$color_secondary]$wg_activated_gh_account #[default]| #[fg=$color_working_dir]$wg_working_dir #[default]| CPU: #{cpu_fg_color}#{cpu_percentage} #[default]| RAM: #{ram_fg_color}#{ram_percentage}  "
```

Comment:
what is this purpose?

Reply:
<img width="114" alt="image" src="https://github.com/jiyeol-lee/dotfiles/assets/97920073/2f2a5305-e484-4b58-aeb6-1c8abfadbc80">
I thought you accidentally deleted that along with the battery settings.
If that is not the case, you don't need to add it.😂

Reply:
oh, you are right. it's a mistake. thank you for the update. feel free to merge

Reply:
😀 Good. Then, can I get write access? I think I'm blocked from merge.....
<img width="918" alt="image" src="https://github.com/jiyeol-lee/dotfiles/assets/97920073/2bbef038-058a-4ae6-8193-52ab898994f8">